server/settings.py

```diff
@@ -202,9 +203,40 @@ def env(variable, fallback_value=None):
 # NTB NITF specific behaviour
 # TODO: move this to a module once SD-4650 is fixed
 
-def add_subjects_scheme(subjects):
-    for subject in subjects:
-        subject['scheme'] = 'subject_custom'
+SUBJECT_TYPE = 'tobject.subject.type'
+SUBJECT_MATTER = 'tobject.subject.matter'
+SUBJECT_DETAIL = 'tobject.subject.detail'
+
+
+def build_subject(xml):
```

Comment:
That's right and this is planed but blocked by SD-4650 (check the comment l.204: # TODO: move this to a module once SD-4650 is fixed).

The issue is that we need to create a subclass with a new parser name (something like ntb-nitf), but everything is hardcoded in client at the moment, so it's temporarily in settings.